courses/free-courses-ml.md

```diff
@@ -182,6 +182,7 @@
 * [PHP Programming Malayalam Tutorial for Beginners](https://www.youtube.com/watch?v=nFYWCouZ1UA) - Yes Tech Media
 * [php tutorial in malayalam](https://youtube.com/playlist?list=PLBKJGko2JEdH_T2ki6ty4xGV19qB7Hpmm) - Tintu Vlogger
 * [Learn PHP programming in malayalam](https://www.youtube.com/playlist?list=PL-Z1WBeTYPOoO9mJxVveZ1VgZPVXmyhFL) - Aks Programming
+* [PHP Programming Course Malayalam](https://www.youtube.com/watch?v=SP4RK8fajFA&list=PLefF9wgiOa4WeDSpmKb6gRS-UJKn2FRnN) - Nikin Sk
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
```suggestion
* [PHP Programming Course Malayalam](https://www.youtube.com/playlist?list=PLefF9wgiOa4WeDSpmKb6gRS-UJKn2FRnN) - Nikin Sk
```